samples/bluetooth/peripheral_fast_pair/src/battery_module.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <bluetooth/services/fast_pair.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_DECLARE(fp_sample, LOG_LEVEL_INF);
+
+#include "battery_module.h"
+
+static const struct bt_fast_pair_battery_comp_desc
+					fp_battery_comp_descs[BT_FAST_PAIR_BATTERY_COMP_COUNT] = {
+	{ .charging = true, .level = 67},
+	{ .charging = false, .level = 100},
+	{ .charging = false, .level = 1}
+};
```

Comment:
We could use enum values to denote indexes (then it would be more readable). See nRF Desktop's LED state definitions for example.